python/packages/autogen-ext/examples/openai_agent_builtin_tools_example.py

```diff
@@ -0,0 +1,231 @@
+"""
```

Comment:
Ideally, this file should be removed.
Please follow convention - examples should be in the docstrings